src/libs/sea_ai/src/locator_show.cpp

```diff
@@ -155,7 +157,7 @@ void SeaLocatorShow::Realize(uint32_t Delta_Time)
     if (core.Controls->GetDebugAsyncKeyState('5') < 0)
     {
         bShow ^= 1;
-        Sleep(200);
+        std::this_thread::sleep_for(std::chrono::milliseconds(100));
```

Comment:
```suggestion
        std::this_thread::sleep_for(std::chrono::milliseconds(200));
```
(constant from the original code)